lib/util.js

```diff
@@ -373,20 +373,37 @@ function _fetch_rss_json (feed_url) {
     });
 }
 
+function _is_wordpress_feed(feed_json) {
+    const has_generator = feed_json.meta.generator &&
+          feed_json.meta.generator.startsWith('https://wordpress.org/');
+    const has_xmlns = feed_json.meta['rss:site'] &&
+          feed_json.meta['rss:site']['@'] &&
+          feed_json.meta['rss:site']['@']['xmlns'] &&
+          feed_json.meta['rss:site']['@']['xmlns'].startsWith('com-wordpress');
+    return has_generator || has_xmlns;
+}
+
 function _fetch_rss_page (feed, page, items, all_items, max_items, oldest_date) {
-    let feed_obj, is_paginated;
-    if (typeof feed === 'function') {
-        is_paginated = true;
-        feed_obj = feed(page);
+    let is_array = false;
+    let is_paginated = false;
+    let feed_url, feed_obj;
```

Comment:
Hmm... we need to figure out if multiple var declarations on a single line is something we want to do but since we don't have rules for it yet, not a blocker.

Reply:
Pick one and add it to eslint :-)

Reply:
We use jshint and it looks like there's no support for this (even though I found some info talking about `mutltivar` which turns out it doesn't work). Maybe we should convert to eslint linting?